src/main/java/de/company/BlackWheel.java

```diff
@@ -0,0 +1,11 @@
+package de.vimo;
+
+public class BlackWheel {
+    int red = 33;
+    int blue = 33;
+    int green = 33;
+
+    public String getBlackWheel() {
```

Comment:
Why does getBlackWheel return a string? 
I would expect it to return a wheel of some kind and not a string.